app/views/shared/_spam_honeypot.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# Honeypot field intended to reduce spam %>
+<div class="visually-hidden" aria-hidden="true">
+  <label for="val">Ignore if you're not a robot</label>
+  <input type="text" tabindex="-1" autocomplete="off" id="val" name="<%= "#{form_name}" %>[val]"/>
```

Comment:
Is there a reason for the `tabindex`? Is it something that spammers might otherwise overlook?
I'm not sure how a screen reader would react to something that on the one hand has a `tabindex` but on the other hand is hidden from them. It's probably fine, but we could also test that to make sure.